kustomize/overlays/stable/shard/patches/deploy-config.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 data:
-  EVENT_BRIDGE_EXECUTOR_IMAGE: quay.io/5733d9e2be6485d52ffa08870cabdee0/executor:133028d5f563c9a4693a04f7f5775a08c225b9c7-jvm
-  EVENT_BRIDGE_INGRESS_IMAGE: quay.io/5733d9e2be6485d52ffa08870cabdee0/ingress:133028d5f563c9a4693a04f7f5775a08c225b9c7-jvm
+  EVENT_BRIDGE_EXECUTOR_IMAGE: quay.io/5733d9e2be6485d52ffa08870cabdee0/executor:68124ea912753f2d2f677fdf3d486d871877ced0-jvm
+  EVENT_BRIDGE_INGRESS_IMAGE: quay.io/5733d9e2be6485d52ffa08870cabdee0/ingress:68124ea912753f2d2f677fdf3d486d871877ced0-jvm
```

Comment:
@lucamolteni Image hash here is `3f9af612c1d92f9c32cba64d79a628770beb8c27` (see [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/dd46981390078614d00095d6551f0928ed791603/kustomize/overlays/prod/shard/patches/deploy-config.yaml#L3-L4))

Reply:
fixed...thx :+1: